examples/models/data_tables_server.py

```diff
@@ -23,10 +27,14 @@ def __init__(self):
         self.class_filter = None
 
         self.source = ColumnDataSource()
+        # A bit hacky, but now we have an index column that will carry through our slicing
+        self.df = DataFrame.from_dict(ColumnDataSource.from_df(mpg))
+        self.df.set_index('index', inplace=True)
         self.update_data()
-
-        self.document.add_root((self.create()))
-        self.session = push_session(self.document)
+        self.source.on_change('data', self.update_from_table_change)
+        # XXX: I haven't had much luck running the server in this style
+        # self.document.add_root((self.create()))
+        # self.session = push_session(self.document)
```

Comment:
is the call to `add_root` still commented out? The output looks fine, it seems like its rendering an empty document possibly. Also to be clear, I would not describe the client version as "standalone" since since an "empty" `bokeh serve` has to be run somewhere. Finally although often technically possible to have one script, since `bokeh serve app.py` will automatically ignore calls to `push_session`, etc, I can't recommend this kind of usage.